HealthMonitoring.SelfHost/packages.config

```diff
@@ -18,6 +18,9 @@
   <package id="Microsoft.Owin.Hosting" version="3.0.1" targetFramework="net451" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net451" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net451" />
+  <package id="Node.js" version="5.3.0" targetFramework="net451" />
+  <package id="NoGit" version="0.1.0" targetFramework="net451" />
+  <package id="Npm" version="3.5.2" targetFramework="net451" />

```

Comment:
Are those 3 package needed to compile or package HealthMonitoring.SelfHost or do they belong to the testing project?
If second is the case, they should be moved out SelfHost packages.config